src/preact/utils.js

```diff
@@ -16,15 +16,31 @@ export function useResourcesNotify() {
 }
 
 /**
- * Combines multiple refs to pass into `ref` prop.
- * @param {...any} refs
- * @return {function(!Element)}
+ * @param {any} ref
```

Comment:
The `any` isn't valid closure 😢 . Should be `*`

Reply:
Updated type to `{current: ?}|function()`